map/framework.cpp

```diff
@@ -1418,11 +1418,7 @@ void Framework::FillSearchResultsMarks(SearchResultsIterT beg, SearchResultsIter
     {
       auto const fID = r.GetFeatureID();
       mark->SetFoundFeature(fID);
-
-      if (r.m_details.m_isHotel)
-        mark->SetHotelType();
-      else
-        mark->SetFromType(r.GetFeatureType());
```

Comment:
Isn't it a regression? Are search results (hotels) highlighted with an appropriate icon?

Reply:
Hotel search results are highlighted with the hotel icon just fine. Instead of using a special case for hotels, I have added the hotel types to the general SetFromType function.